### ISSUE TITLE
Classic: missing timers after update

### ISSUE THREAD

**Issue by c0mm0n** [BigWigsMods/Capping#64]
Hey i had Capping with the modified Battlegrounds.lua from another ticket working fine.

Today I updated, and now I miss some timers

<img width="618" alt="image" src="https://user-images.githubusercontent.com/1181493/72454763-c9ad1200-37c1-11ea-9866-275982296cfb.png">

Should i get back the modified .lua ?

**Comment by c0mm0n** [comment]
<img width="636" alt="image" src="https://user-images.githubusercontent.com/1181493/72454840-efd2b200-37c1-11ea-9ab0-f4b03d663061.png">

**Comment by c0mm0n** [comment]
<img width="617" alt="image" src="https://user-images.githubusercontent.com/1181493/72480127-ae102e80-37f6-11ea-9e91-3560f532b493.png">

Also this

The hut was not even being capped (taken by alliance)

**Comment by phendrome** [comment]
Yup. They're completely messed up after the update. The `battlegrounds.lua` fix in the old version was perfect.
